qcoverage/src/org/qcmg/coverage/Coverage.java

```diff
@@ -70,7 +68,7 @@ private void writePerFeatureTabDelimitedCoverageReport( final QCoverageStats sta
 	}
 
 	private void writePerTypeTabDelimitedCoverageReport(final QCoverageStats stats) throws IOException {
-		final File file = new File(options.getOutputFileNames()[0]);
+		final File file = new File(options.getOutputFileNames()[0]+ ".txt");
```

Comment:
what if the output file name already has a ".txt" extension?

Reply:
I think *.txt.txt is OK for text output, *.txt.xml is better than *.txt for XML output, etc

Reply:
I think adding code to make this smarter is a good idea

Reply:
function added. However, our WDL pipeline has to be smart as well to predict the output file name.